functions/src/utils/permissions.ts

```diff
@@ -62,4 +138,40 @@ export const checkIsWorldOwner = async (worldId: string, uid: string) => {
   }
 };
 
-exports.checkIsAdmin = checkIsAdmin;
+export const throwErrorIfNotWorldOwner = async ({
+  worldId,
+  uid,
+}: {
+  worldId: string;
+  uid: string;
+}) => {
+  const isWorldOwner = await checkIfWorldOwner({ worldId, uid });
+
+  if (!isWorldOwner) {
+    throw new HttpsError(
+      "permission-denied",
+      `User is not an owner of ${worldId}`
+    );
+  }
+};
+
+export const throwErrorIfNeitherWorldNorSpaceOwner = async ({
+  worldId,
+  spaceId,
+  uid,
+}: {
+  worldId: string;
+  spaceId: string;
+  uid: string;
+}) => {
+  console.log("I'm inside");
```

Comment:
Stray console log

Reply:
Removed in https://github.com/sparkletown/sparkle/pull/2820/commits/af825f995a7267c0da1cdbc23ea8630055576519